tpl/toolbox/report.tpl.php

```diff
@@ -64,6 +64,25 @@
 					<textarea id="litespeed-report" rows="20" cols="100" readonly><?php echo esc_textarea($report); ?></textarea>
 				</td>
 			</tr>
+			<tr>
+				<th>&nbsp;</th>
+				<td>
+					<?php
+					$this->build_checkbox(
+						'attach_php',
+						sprintf(
+							__(
+								'Attach PHP info to report. Check this box to insert the %1$s sections from %2$s.',
+								'litespeed-cache'
+							),
+							'<code>INFO_GENERAL</code>, <code>INFO_CONFIGURATION</code> and <code>INFO_MODULES</code>',
```

Comment:
`and` shouldn't be out of translation. Plz join these 3 w/ `/`.

Reply:
Grammatically, I'm not a fan of `/`. Can you do something like this? (not sure about the syntax)
```
	'Attach PHP info to report. Check this box to insert the %1$s and %2$s sections from %3$s.',
								'litespeed-cache'
							),
							'<code>INFO_GENERAL</code>, <code>INFO_CONFIGURATION</code>',
							'<code>INFO_MODULES</code>',
							'<a href="https://www.php.net/manual/en/function.phpinfo.php" target="__blank">phpinfo()</a>'
```
(BTW, you had `target` misspelled as `traget` in the last line)

Reply:
@hi-hai 
I thought to keep them in translation, but if we change the parameters(add/remove flags) we will not need to change text translation

the split was @lslisa  ideea. I am ok changing them  to 1 code and /  between, but the change was suggested by her

Reply:
If we can avoid using `/` and keep the `and` in there by using an additional parameter like I suggested, that would be nicer.

Reply:
Those consts explanation are unnecessary to the end users. Maybe we can just remove them and simply say phpinfo?

Reply:
@hi-hai like this?